lib/src/git_backend.rs

```diff
@@ -907,6 +947,8 @@ impl Backend for GitBackend {
     }
 
     fn write_commit(&self, mut contents: Commit) -> BackendResult<(CommitId, Commit)> {
+        assert!(contents.secure_sig.is_none(), "commit.secure_sig was set");
```

Comment:
nit: this should belong to the next commit

Reply:
sure